### PR TITLE
Update nf-core to 1.10.2

### PR DESCRIPTION
**Pull request by BiocondaBot**
Update [`nf-core`](https://bioconda.github.io/recipes/nf-core/README.html): **1.9** &rarr; **1.10.2**

[install with bioconda](http://bioconda.github.io/recipes/nf-core/README.html) [Conda](https://anaconda.org/bioconda/nf-core/files)

Info | Link
-----|-----
Recipe | [`recipes/nf-core`](https://github.com//bioconda/bioconda-recipes/tree/bump/nf_core/recipes/nf-core) (click to view/edit other files)
Releases |[https://pypi.org/pypi/nf-core/json](https://pypi.org/pypi/nf-core/json)

***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

File: recipes/nf-core/meta.yaml [hunk 1]
```diff
@@ -30,7 +30,9 @@ requirements:
     - requests-cache
     - tabulate
     - git
-    - jsonschema 
+    - jsonschema
+    - rich
+    - inquirer
```
Comment (ewels) [inline recipes/nf-core/meta.yaml:35]: Sadly `inquirer` != `pyinquirer`. Looks like `pyinquirer` might not be on conda yet..?

`setup.py` diff between releases [here](https://github.com/nf-core/tools/compare/1.9...master#diff-2eeaed663bd0d25b7e608891384b7298)

New requirements: 

* jinja2
* PyInquirer==1.0.2
* rich>=4.0.0

```suggestion
    - jinja2
    - rich >=4.0.0
    - pyinquirer=1.0.2
```

**Comment by apeltzer** [comment]
Waiting for this here to be fine: https://github.com/conda-forge/staged-recipes/pull/12268

**Comment by apeltzer** [comment]
Almost ready now! :-) https://github.com/conda-forge/staged-recipes/pull/12268

**Comment by apeltzer** [comment]
@bioconda-bot please merge

**Comment by ewels** [comment]
Nice - thanks @apeltzer!

**Pull request merged**